xml/ay_auto_installation.xml

```diff
@@ -280,6 +280,19 @@ xml:id="Invoking">
        Location of the installation directory, for example
        <literal>install=nfs://192.168.2.1/CDs/</literal>.
       </para>
+      <note>
+       <title>Disabling SSL checks</title>
+       <para>
+         When you are using HTTPS, SSL checking is enabled by default. If 
+         necessary, you can disable SSL checking by appending <literal>?ssl_verify=no</literal>
```

Comment:
Um, actually appending the `?ssl_verify=no` text is not correct in all cases, if there already is an option in the URL you should append the `&ssl_verify=no` text. The `?` character separates the URL query from path, the `&` character separates multiple query options. That's the usual [URL format](https://en.wikipedia.org/wiki/URL).

E.g. `https://example.com?foo=bar` => `https://example.com?foo=bar&ssl_verify=no`

So I'd rephrase it as

```
... by appending the <literal>ssl_verify=no</literal> query option ...
```

Reply:
Good catch, thanks!